src/main/java/com/projetonetflix/projetonetflix/controller/GenreController.java

```diff
@@ -29,4 +31,11 @@ public List<GenreDTO> getGenres(){
         return listDTO;
     }
 
+    @GetMapping(path = "{name}")
+    public List<GenreDTO> getByNameIgnoreCase(@PathVariable String name) {
+        List<Genre> listDirector = genreService.findByNameIgnoreCase(name);
```

Comment:
isso aqui funciona quando você busca o nome inteiro exato, ou também serve quando você coloca só um teco?
você busca por anéis. e vem senhor do anéis e tudo mais que tiver anéis, ou só pega exato?
exemplo: https://www.imdb.com/find?q=rings&s=tt&ref_=fn_al_tt_mr

Reply:
Ele vai pegar o nome exato, porem é vai ignorar se é escrito com letra maiúscula ou minúscula, acho melhor alterar para pegar contendo.